diktat-common/src/main/kotlin/org/cqfn/diktat/common/config/rules/RulesConfigReader.kt

```diff
@@ -84,15 +84,22 @@ open class RulesConfigReader(override val classLoader: ClassLoader) : JsonResour
     }
 }
 
-fun List<RulesConfig>.getCommonConfiguration() = lazy { CommonConfiguration(getCommonConfig()) }
+/**
+ * @return common configuration from list of all rules configuration
+ */
+fun List<RulesConfig>.getCommonConfiguration() = lazy { CommonConfiguration(getCommonConfig()?.configuration) }
 
-class CommonConfiguration(commonConfig: RulesConfig?) {
+/**
+ * class returns the list of common configurations that we have read from a configuration map
+ * @param configuration map of common configuration
+ */
+open class CommonConfiguration(configuration: Map<String, String>?) {
```

Comment:
```suggestion
class CommonConfiguration(configuration: Map<String, String>?) {
```
I think for now we don't need this class to be open.

Reply:
https://github.com/cqfn/diKTat/pull/271/checks?check_run_id=1116794157 - still some failures on diktat-common